config/defaults.conf

```diff
@@ -216,6 +216,8 @@ bert_embeddings_mode = "none"  // How to handle the embedding layer of the
                                // "cat" returns top-layer concatenated with
                                //   lexical layer
                                // "only" returns only lexical layer
+                               // "mix" uses ELMo-style scalar mixing (with
+                               // learned weights) across all layers
```

Comment:
Make it clear what one should do to run BERT-style fine-tuning?

Reply:
That's handled by the `bert_fine_tune` parameter below.

Reply:
Er, sorry: What should one set here to follow the standard BERT protocol (when bert_fine_tune = 1). None? Does it not matter?

Reply:
Clarify and merge?